.circleci/config.yml

```diff
@@ -97,34 +117,35 @@ jobs:
       - slack/notify:
           message: "Orb publishing was approved for any orbs modified in commit ${CIRCLE_SHA1} on the ${CIRCLE_BRANCH} branch: https://circleci.com/workflow-run/${CIRCLE_WORKFLOW_ID}"
           mentions: "${ROSE_SLACK_UUID},${EDDIE_SLACK_UUID}"
+      - circle-compare-url/reconstruct
 
       - run:
           name: Publish any modified orbs
           shell: /bin/bash -exo pipefail
           command: |
-            GIT_LOG=$(git log -1 --format="" --name-only)
+            # save value stored in file to a local env var
+            CIRCLE_COMPARE_URL=$(cat CIRCLE_COMPARE_URL)
+
+            COMMIT_RANGE=$(echo $CIRCLE_COMPARE_URL | sed 's:^.*/compare/::g')
 
-            echo "Files changed:" $GIT_LOG
+            echo "Commit range: $COMMIT_RANGE"
 
             for ORB in src/*/; do
               orbname=$(basename $ORB)
-
-              if [[ $(git log -1 --format="" --name-only | grep "$orbname") ]]; then
-
+              if [[ $(git diff $COMMIT_RANGE --name-status | grep "$orbname") ]]; then
                 echo "promoting circleci/${orbname}@dev:${CIRCLE_BRANCH}-${CIRCLE_SHA1} as patch release"
-
                 circleci orb publish promote circleci/${orbname}@dev:${CIRCLE_BRANCH}-${CIRCLE_SHA1} patch
```

Comment:
Do we need to do a check for the api token here as well?

Reply:
i updated it to just pass in the token every time @bradylill

Reply:
see related convo here : https://github.com/CircleCI-Public/orb-tools-orb/pull/6